src/main/java/gmailgdogra/OutputRow.java

```diff
@@ -0,0 +1,35 @@
+package gmailgdogra;
+
+public class OutputRow {
+    private final String firstName;
+    private final String lastName;
+    private final String swipeTime;
+    private final String deviceName;
+
+    public static OutputRow of(String firstName, String lastName, String swipeTime, String deviceName) {
```

Comment:
If you use this factory method, it'd be a good idea to add assertions that the args are not null

Reply:
agree